Source/WEB_ROOT/admin/asset/scripts/getAssetsHelper.html

```diff
@@ -87,13 +87,14 @@
 			
 		FROM u_assettable asset
 		cross join u_assetsettings s
-		WHERE ~[if#schoolCheck.is.a.school]asset.SchoolName = '~(schoolname)' [else#schoolCheck]SchoolName IS NOT null  [/if#schoolCheck] 
+		WHERE 
+		--~[if#schoolCheck.is.a.school]asset.SchoolName = '~(schoolname)' [else#schoolCheck]SchoolName IS NOT null  [/if#schoolCheck] 

```

Comment:
There's little you can do here, but it would be really nice to separate concerns in a more traditional stack. I would throw this logic at least in a razor view, or if available, down into a business class that utilizes a view model to instruct the view on how to act. That being said, awesome stuff John.